lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -291,6 +291,18 @@
             }
             aboutSectionsData.push(socialFields);
 
+            // Add account deletion fields
+            if (enableGDPRFlag) {
+                accountDeletionFields = {
+                    title: gettext('Delete My Account'),
+                    fields: [],
+                    // Used so content can be rendered external to Backbone
+                    domHookId: 'account-deletion-container'
```

Comment:
Is there some constants file that both this and the `Initializer` could read from instead of having to remember both of these values whenever we need to change the `id`?